### ISSUE TITLE
No errors or warnings displayed in console

### ISSUE THREAD

**Issue by voydz** [webpack-contrib/stylelint-webpack-plugin#77]
Hello,
first of all, thanks for your great work with this plugin.

Lately I am encountering exact the same issue like mentioned in #50 (fixed in #51 with version ^0.5) but with version ^0.7 (latest) of the plugin. 

There are no errors or warnings displayed in the console. Downgrading to ^0.5 fixed the issue. I am using laravel-mix and the config for the plugin looks like this:

```js
new StyleLintPlugin({
    failOnError: false,
    configFile: '.stylelintrc',
    context: 'resources/assets/sass/',
    files: '**/*.scss',
    syntax: 'scss'
}
```

Someone a clue, what the issue could be?

Thanks

**EDIT:**

After further investigation and rolling versions between 0.5 and 0.7. Finally, I read the changelog of the releases. Playing around with `emitErrors` and the `quiet` config options.

**EDIT2:** I had to set `quiet: false` explicitly, though the docs mention that it is the default value. So the lint results are printed at the "start" of the webpack build.

But still, no errors are emitted after the build process finished. If I switch `emitErrors: false` then all errors are suddenly shown, but just as warnings.

Maybe someone could have a look into this?

**Comment by voydz** [comment]
Hey, any chance to get an update on this? Even a workaround would be much appreciated. :)

At the moment all errors are thrown as warnings, resulting in a successful build.  I'd like to fail the build in case an error occurs, while being able to actually see the error.

Thanks in advance

**Comment by JaKXz** [comment]
@voydz do you think that you could submit a failing test in a PR? I'm not sure how to reproduce your issue and I think the cases you're mentioning are covered in the tests, but I could be wrong.

**Comment by shellscape** [comment]
Closing as stale due to age and inactivity. If there is still interest in this issue, please do submit a pull request.
